foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/internal/oxm/util/NamespaceResolverStorage.java

```diff
@@ -0,0 +1,159 @@
+package org.eclipse.persistence.internal.oxm.util;
```

Comment:
Please move to NamespaceResolverStorage to the org.eclipse.persistence.internal.oxm package.